src/typeahead/test/typeahead.spec.js

```diff
@@ -1049,15 +1071,15 @@ describe('typeahead deprecation', function() {
     expect($log.warn.calls.argsFor(1)).toEqual(['typeahead is now deprecated. Use uib-typeahead instead.']);
     expect($log.warn.calls.argsFor(2)).toEqual(['typeahead-popup is now deprecated. Use uib-typeahead-popup instead.']);
   }));
-  
+
```

Comment:
> Also, sorry for the whitespace changes, I'll correct those once I have working tests. WebStorm will just mess this up again while I'm working on the files."

This is not quite what I had in mind when I was asking for help with the tests ;)